src/app/WriteHandler.h

```diff
@@ -76,6 +76,11 @@ class WriteHandler : public Messaging::ExchangeDelegate
      */
     void Abort();
```

Comment:
A followup: we should just remove `Abort`.  It's only used from `InteractionModelEngine::Shutdown`, which could just as easily check `IsFree` and call `Close`.

Reply:
https://github.com/project-chip/connectedhomeip/issues/21739

Reply:
https://github.com/project-chip/connectedhomeip/issues/21740